custom_item_loader/src/CustomItem.cs

```diff
@@ -91,6 +92,51 @@ public CustomItem(
 		};
 	}
 
+	/// <summary>
+	/// This sets up the "gadget" components required
+	/// </summary>
+	/// <param name="itemPrefab"></param>
+	/// <exception cref="NotImplementedException"></exception>
```

Comment:
We're not throwing a NotImplementedException, this line can be removed

Reply:
This has been removed